front/app/components/UI/QuillEditedContent/QuillEditedContent.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen, waitFor } from 'utils/testUtils/rtl';
+import QuillEditedContent from '.';
+import 'jest-styled-components';
+
+describe('QuillEditedContent component', () => {
+  it('Should change tabIndex on children properly', async () => {
+    // multiple rerenders to simulate a user showing/hiding the contentnt multiple times
+
+    const { rerender } = render(
+      <QuillEditedContent>
+        <a href="www.google.com" data-testid="link">
+          link
+        </a>
+        <button data-testid="button">button</button>
+        <iframe data-testid="iframe">
+          <div>iframe content</div>
+        </iframe>
+      </QuillEditedContent>
+    );
+
+    rerender(
```

Comment:
I don't think this first re-render is needed - good to test with undefined as well.

Reply:
Alright, let's leave this as is in this case :)